src/core/public/_mixins.scss

```diff
@@ -1,6 +1,6 @@
 @mixin kibanaFullBodyHeight($additionalOffset: 0) {
-  // The `--euiFixedHeadersOffset` CSS variable is automatically updated by
+  // The `--kbnAppHeadersOffset` CSS variable is automatically updated by
   // styles/rendering/_base.scss, based on whether the Kibana chrome has a
-  // header banner, and is visible or hidden
-  height: calc(100vh - var(--euiFixedHeadersOffset, 0) - #{$additionalOffset});
+  // header banner, app menu, and is visible or hidden
+  height: calc(100vh - var(--kbnAppHeadersOffset, 0) - #{$additionalOffset});
```

Comment:
This mixin is used in couple places. In particular this change fixes the full screen page layout in discover in serverless